front/app/components/PostShowComponents/Comments/CommentingDisabled.tsx

```diff
@@ -46,7 +46,10 @@ const calculateMessageDescriptor = (
     return messages.commentingDisabledUnverified;
   } else if (isLoggedIn && commentingDisabledReason === 'not_permitted') {
     return messages.commentingDisabledProject;
-  } else if (isLoggedIn && commentingDisabledReason === 'not_active') {
+  } else if (
+    (isLoggedIn && commentingDisabledReason === 'not_active') ||
+    commentingDisabledReason === 'missing_data'
+  ) {
```

Comment:
Do we still need to support the "not_active" disabled reason? :thinking:

Reply:
I think so, although the only scenario for this I think is that the user will be blocked, so they shouldn't be able to login anyway. I'd leave it in anyway.